commands.go

```diff
@@ -1677,16 +1677,22 @@ func (c *commandable) GeoAdd(key string, geoLocation ...*GeoLocation) *IntCmd {
 }
 
 func (c *commandable) GeoRadius(query *GeoRadiusQuery) *GeoLocationCmd {
```

Comment:
I've exposed 2 different methods now, but am undecided about the new method signature - one looks cleaner but the other aligns more with Redis itself....